templates/example_pixel_cell_clustering.ipynb

```diff
@@ -0,0 +1,679 @@
+{
```

Comment:
<p>Same here, these can be abstracted and removed from notebook with a link to documentation if people want to tweak them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>